doc/whats_new/v1.0.rst

```diff
@@ -608,6 +608,10 @@ Changelog
 :mod:`sklearn.manifold`
 .......................
 
+- |Fix| Decrease the numerical default tolerance in the lobpcg call
+  in :func:`manifold.spectral_embedding` to prevent numerical instability. 
+  :pr:`21194` by :user:`Andrew Knyazev <lobpcg>`.
```

Comment:
Please move this entry after enhancement to respect the canonical ordering of issue types defined in https://scikit-learn.org/stable/whats_new/v1.0.html#legend-for-changelogs.